mreg_cli/commands/network.py

```diff
@@ -314,19 +269,14 @@ def find(args: argparse.Namespace) -> None:
     :param args: argparse.Namespace (limit, silent, addr_only, ip, network, description, vlan,
                                      dns_delegated, category, location, frozen, reserved)
     """
-    return
     args_dict = vars(args)
 
-    ip_arg = args_dict.get("ip")
-
-    if ip_arg:
-        ip_range = get_network_range_from_input(ip_arg)
-        network_info = get_network(ip_range)
-        if not network_info:
-            cli_warning(f"No network found for ip {ip_arg}")
-        networks = [network_info]
+    if ip_arg := args_dict.get("ip"):
+        # We technically support ID args when calling this, but that's fine.
+        net = Network.get_by_any_means_or_raise(ip_arg)
+        networks = [net]
```

Comment:
The question is: Why did this support network addresses in the first place? We should probably just remove the support for that and change this to `Network.get_by_ip_or_raise`.

Reply:
Network and ID arg support removed in [f5642ff](https://github.com/unioslo/mreg-cli/pull/238/commits/f5642ff6ca524b28dd2d41c4e7867f1893ef5215)